src/Compilers/CSharp/Test/Semantic/Semantics/InterceptorsTests.cs

```diff
@@ -12,46 +13,30 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests.Semantics;
 
 public class InterceptorsTests : CSharpTestBase
 {
-    // PROTOTYPE(ic): test a case where the original method has type parameter constraints.
-    // PROTOTYPE(ic): for now we will just completely disallow type parameters in the interceptor.
-    // PROTOTYPE(ic): test where interceptable is a constructed method or a retargeting method.
     // PROTOTYPE(ic): Ensure that all `MethodSymbol.IsInterceptable` implementations have test coverage.
-    // PROTOTYPE(ic): test where there are differences between 'scoped' modifiers and '[UnscopedRef]' attributes
 
     // PROTOTYPE(ic): Possible test cases:
     //
     // * Intercept instance method with instance method in same class, base class, derived class
     // * Intercept with extern method
-    // * Intercept with abstract or interface method
     // * Intercept an abstract or interface method
     // * Intercept a virtual or overridden method
     // * Intercept a non-extension call to a static method with a static method when one or both are extension methods
     // * Intercept a struct instance method with an extension method with by-value / by-ref this parameter
     // * An explicit interface implementation marked as interceptable
-    // * Intercept a generic method call when the type parameters are / are not substituted
 
-    // PROTOTYPE(ic): test a valid case with large numbers of interceptors. (EndToEndTests?)
```

Comment:
>test a valid case with large numbers of interceptors. (EndToEndTests?) [](http://example.com/codeflow?start=22&length=70)

Are we testing a large number of interceptors (and intercepted calls), to ensure there are no obvious perf issues with resolving intercepted calls?

Reply:
Not yet. I should have noted that some of these were moved to the test plan, but not checked off yet.